thinkshout/thinkwp-starter-theme
# 77 base assets

#77 This PR removes base-assets files from `thinkwp-starter-theme`, and adds a composer script to bring them back in from the `[thinkshout/base-assets](https://github.com/thinkshout/base-assets)` repo.

I added a new composer script file to the theme itself. This also works if we want to add the script to the top-level composer.json file. 

If adding it to the top level composer.json file, the script needs to be changed slightly, and the theme name would need to use a variable instead of pointing to think-friends:

``` 
"scripts": {
    "scaffold": [
      "git clone git@github.com:thinkshout/base-assets.git",
      "mv base-assets/scaffold/* web/wp-content/themes/custom/think-friends/",
      "rm -rf base-assets"
    ]
  }
```

## To test:
- pull in this branch
- From the theme, run `composer scaffold`
- Expect all files and directories in `scaffold` from base-assets to appear in the theme in the correct locations. 
- Run `npm install`, and `npm run build`
- Expect /style-guide to be populated with base build atoms, molecules and blocks. 
- Expect the site nav and footer to be populated with the base build styles, including mobile nav
